src/openrct2.c

```diff
@@ -59,6 +59,7 @@ utf8 gCustomPassword[MAX_PATH] = { 0 };
 bool gOpenRCT2Headless = false;
 
 bool gOpenRCT2ShowChangelog;
+bool gOpenRCT2SilentBreakpad;
```

Comment:
I think I tried, but I may have mucked things when it comes to `extern "C" {}` stuff and I don't really have VS handy right now to check this, so I have to rely on appveyor. Feel free to improve that.